simulator/modules/execute/execute.cpp

```diff
@@ -123,7 +124,10 @@ void Execute<ISA>::clock( Cycle cycle)
         wp_bypass->write( instr.get_dst_v(), cycle);
 
         if ( instr.is_mem_stage_required())
+	{
```

Comment:
Please, do not use tabs for indentation here and everywhere.

Reply:
Please double-check for tabs and set up your editor: https://github.com/MIPT-ILab/mipt-mips/wiki/Code-style-guidelines#indentation